packages/hardhat-chai-matchers/test/bigNumber.ts

```diff
@@ -877,20 +877,32 @@ describe("BigNumber matchers", function () {
 
           // failed assertions
           expect(() => expect([a]).to.not.deep.equal([b])).to.throw(
-            AssertionError
+            AssertionError,
+            // [\s\S]* is used here instead of .* so that newlines are included
```

Comment:
Is there a reason not to use [the `s` flag](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#advanced_searching_with_flags)?

Reply:
Oh, great catch, thank you. Fixed in 1e5bc1c53a09ceb16171cee195c8ea41ae6bb062.